server/data/frontendComponentApiClient.test.ts

```diff
@@ -0,0 +1,36 @@
+import nock from 'nock'
+import config from '../config'
+import FrontendComponentApiClient from './frontendComponentApiClient'
+
+describe('frontendComponentApiClient', () => {
+  const frontendComponentApiClient = new FrontendComponentApiClient()
+
+  config.apis.frontendComponents.url = 'http://localhost:8200'
+  let fakeFrontendComponentApi: nock.Scope
+
+  beforeEach(() => {
+    fakeFrontendComponentApi = nock(config.apis.frontendComponents.url)
+  })
+
+  afterEach(() => {
+    nock.cleanAll()
+  })
+
+  describe('getComponent', () => {
+    it('should return data from api', async () => {
+      // Given
+      const userToken = 'a-user-token'
+
+      const expectedResponse = { data: 'data' }
```

Comment:
Could you create a test data builder in `testsupport` please that represents a real response please? Whilst the test itself is nice and easy to read, I can't grok what the response data "looks" like from this test, so would be hard for someone else to maintain the code and make future changes (eg:. from reading this test I might incorrectly assume the response from the frontend components API was an object with a single field `data`)

Something like:
```
const aValidFrontEndComponentsFooterResponse = (): FrontEndComponentResponse {
    html: '<p>some html</p>',
    css: ['https://frontend-componenents-dev/footerStyles.css'],
    js: ['https://frontend-componenents-dev/footerScripts.js'],
}

```

Reply:
Added a `aValidFrontEndComponentFooter` response and used it within the test